### DISCUSSION TITLE
Web game

### DISCUSSION THREAD

**Discussion by laparda14** [RighteousRyan1/WiiPlayTanksRemake#52] [Ideas]
Could you make it so it is playable on a webpage?

**Comment by RighteousRyan1** [comment]
This game probably isn't lightweight enough for a browser to handle it,
plus 500MB is the maximum memory a web page can consume. This game can
sometimes go over that, since the .NET runtime is hard to optimise
memory-wise. I'd have to find/make a conversion framework from
OpenGL/DirectX11/9 to something like WebGL. It might exist, but then I'd
have to pay to host the game on a server, which is something I can't
currently fund, especially for a game like this, unfortunately.

If there's any other questions you want to ask and have answered, feel free
to ask them.

**Reply by laparda14** [reply]
Ok I have it on a windows computer, but I can’t play it on an iPad. If it was possible, I could run it on an iPad? I’m just wondering.

**Comment by laparda14** [comment]
I hate to ask, but is Ryan your real first name?
Plus, you are now a senior in high school?! I’m just a sophomore now(2024). Wow!

**Comment by laparda14** [comment]
Did you see tooni’s video on YouTube with you, him, soverein, and bigkitty playing this multiplayer game? You said you would sue tooni if he included something in his video. He included that in it.

**Reply by RighteousRyan1** [reply]
It was satirical.

**Reply by laparda14** [reply]
Lol

**Reply by laparda14** [reply]
Did you fix the bugs yet?

**Reply by RighteousRyan1** [reply]
What are you talking about?

**Reply by laparda14** [reply]
The bug on one of the levels that doesn’t let the shell bounce

**Reply by RighteousRyan1** [reply]
nope

**Comment by laparda14** [comment]
Maybe I could help?
By: Damian Laparr

On May 28, 2024, at 9:10 PM, RighteousRyan ***@***.***> wrote:

﻿

nope
